.github/workflows/build-rc.yml

```diff
@@ -20,6 +20,9 @@ on:
         required: true
         type: string
 
+env:
+  EAS_PROJECT_URL: 'https://expo.dev/accounts/digidem/projects/example-app'
```

Comment:
felt helpful to represent this as a variable instead of hardcoding it where it's used, for the sake of making integration into other projects easier.

maybe this should be a repo variable?

another thing that I haven't figured out is if there's a way to get this URL via the EAS CLI. Haven't found anything yet...